databroker/assets/tests/test_fs_mutate.py

```diff
@@ -232,3 +233,16 @@ def local_handler(rpath):
         path = fs.retrieve(dm['datum_id'])
 
     assert path == os.path.join('baz', 'foo')
+
+    # test two root maps work
+    res = fs.insert_resource('root-test', 'foo', {}, root='baz2')
```

Comment:
should this be `bar2`?

Reply:
oops yes. tests passed but to be consistent with the logic, it should be `bar2`. thanks!